.github/workflows/actions.yml

```
@@ -5,15 +5,68 @@ on: push
 jobs:
   build-and-tests:
     runs-on: ubuntu-latest
+    env: 
+      HAS_TOKEN: ${{ secrets.DOCKERHUB_USERNAME != '' &&  secrets.DOCKERHUB_TOKEN != ''}}
     container:
       image: willallves/grpc-build:1.34.0
```

doesn't this need to change as well?
```suggestion
      image: cspiffe/grpc-build:1.34.0
```